client/src/document.js

```diff
@@ -134,6 +125,23 @@ function SidebarLeaf({ depth, title, content }) {
   );
 }
 
+function SidebarLeaflets({ node }) {
+  return (
+    <details>
+      <summary>{node.title}</summary>
```

Comment:
Can you look at adding something like this:
```css
details summary {
  cursor: pointer;
}
```
or something because it's kinda hard to understand that you can actually click on it because when I hover over it nothing happens the cursor is a regular caret thing.